heareval/predictions/task_predictions.py

```diff
@@ -709,15 +709,40 @@ def dataloader_from_split_name(
     metadata: bool = True,
     batch_size: int = 64,
 ) -> DataLoader:
-    dataset = SplitMemmapDataset(
-        embedding_path=embedding_path,
-        label_to_idx=label_to_idx,
-        nlabels=nlabels,
-        split_name=split_name,
-        embedding_type=embedding_type,
-        in_memory=in_memory,
-        metadata=metadata,
-    )
+    """
+    Get the dataloader for a `split_name` or a list of `split_name`
+
+    For a list of `split_name`, the dataset for each split will be concatenated.
+    The dataloader will be built from this concatenated
+    dataset
```

Comment:
explain the two cases with *examples*, so people understand why we need this complexity